core/cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/consent/ConsentProperties.java

```diff
@@ -117,7 +131,74 @@ public void setMongo(final MongoDb mongo) {
     public static class Jpa extends AbstractJpaProperties {
         private static final long serialVersionUID = 1646689616653363554L;
     }
+    
+    public static class Ldap extends AbstractLdapProperties {
+        private static final long serialVersionUID = 1L;
```

Comment:
Review looks great, thanks. I saw you added a "real" UID for the repository. I didn't really know where to get one from, but here it's the same thing I believe.

Reply:
Excellent, thanks!

For the most part, you can "fake" one yourself by copying an existing UID and changing values...or I believe both eclipse and IDEA are able to auto-generate them for you. (IDEA has an inspection in fact where it fails the build if a serializable class does not have UID, or a generated UID I should say).